src/treetensornetworks/opsum_to_ttn/qnarrelem.jl

```diff
@@ -1,21 +1,22 @@
-using StaticArrays: MVector
 
 #####################################
 # QNArrElem (sparse array with QNs) #
 #####################################
 
-struct QNArrElem{T,N}
-  qn_idxs::MVector{N,QN}
-  idxs::MVector{N,Int}
+struct QNArrElem{T}
+  qn_idxs::Vector{QN}
+  idxs::Vector{Int}
```

Comment:
This change made it possible to have a dictionary of vectors of `QNArrElem`'s have a concrete eltype (of each vector). Otherwise the value type was just `Vector`.

Reply:
I don't understand that, could you explain that more?